lib/node-definitions.ts

```
@@ -64,150 +62,62 @@ interface NodeDefinition<TSubType extends string = string> {
-  // Special case for EmailNode - use the new definition
-  if (data.nodeType === NodeType.ACTION && (data as { actionType: ActionType }).actionType === ActionType.EMAIL) {
-    return EMAIL_NODE_DEFINITION as unknown as NodeDefinition
-  }
-  
-  // Use legacy system for other nodes
+
+  // Use the new registry system for all nodes
   switch (data.nodeType) {
     case NodeType.ACTION:
-      return NODE_DEFINITIONS.find((d) => d.nodeType === NodeType.ACTION && d.subType === (data as { actionType: ActionType }).actionType)
+      const actionType = (data as { actionType: ActionType }).actionType
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)